specs-go/config.go

```diff
@@ -22,11 +22,24 @@ type Spec struct {
 	Hooks Hooks `json:"hooks"`
 	// Annotations is an unstructured key value map that may be set by external tools to store and retrieve arbitrary metadata.
 	Annotations map[string]string `json:"annotations,omitempty"`
-
+	// KernelConfig specifies kernel-related configuration for virtual-machine-based runtimes.
+	KernelConfig KernelConfig `json:"kernelConfig"`
+	// ImagePath is the absolute path to the root filesystem image on the host which can be used by a virtual machine-based runtime.
+	ImagePath string `json:"imagePath"`
 	// Linux is platform specific configuration for Linux based containers.
 	Linux Linux `json:"linux" platform:"linux"`
```

Comment:
May be `Linux` should be renamed to `LinuxContainer` & then VM based options can be added as `LinuxVirtualMachine`..

cc @crosbymichael @wking